packages/modules/viewer-react/src/utilities/index.ts

```diff
@@ -15,7 +15,7 @@ export * from "./MakeCancellable";
  * @returns
  */
 export const isObject = (item: any) => {
-  return Object.prototype.toString.call(item) === "[object Object]";
+  return item !== null && typeof item === "object";
```

Comment:
this will return true for arrays, which we don't want

Reply:
good catch